routes/applications.js

```diff
@@ -0,0 +1,35 @@
+//WIP
+const express = require("express");
+const pool = require("../db");
+const router = express.Router();
+router.use(express.json());
+
+router.get("/applications", async (req, res) => {
+    try {
+        const result = await pool.query("SELECT * FROM applications");
+        res.json(result.rows);
+    } catch (err) {
+        console.error(err);
+        res.status(500).send("Server Error");
+    }
+});
+
+
+
+router.post("/applications", async (req, res) => {
+    const { userId, text } = req.body;
```

Comment:
Now I think we need a separate endpoint for creating application to become "member" and creating application to become "mentor". If this is for "member" then I think we won't have the userId, but we'll have other fields completed by user, i.e. firstName, lastName, email etc.